crates/matrix-sdk-sqlite/src/crypto_store.rs

```diff
@@ -767,9 +826,9 @@ impl CryptoStore for SqliteCryptoStore {
         let session_id = self.encode_key("inbound_group_session", session_id);
         let Some((room_id_from_db, value)) =
             self.acquire().await?.get_inbound_group_session(session_id).await?
-        else {
-            return Ok(None);
-        };
+            else {
+                return Ok(None);
+            };
```

Comment:
Why change indentation here?

Reply:
Not sure what happened there and why `cargo fmt` won't help. Reverted in: 6fd129dcfaa501636e2f23e9381ad2fbad06e0ea.